aten/src/ATen/native/vulkan/VulkanConvolution.cpp

```diff
@@ -66,14 +66,13 @@ ContextConv2D create(
   const auto stride_expanded = expand_param_if_needed(stride, "stride", 2);
   const auto dilation_expanded =
       expand_param_if_needed(dilation, "dilation", 2);
-  const Tensor weight_nchw = weight.contiguous();
+  Tensor weight_nchw = weight.contiguous();
+  auto ws = weight_nchw.sizes();
   return ContextConv2D{
-      at::native::vulkan_convolution_prepack_weights(weight),
+      groups == 1 ? at::native::vulkan_convolution_prepack_weights(weight_nchw)
+                  : std::move(weight_nchw.vulkan()),
```

Comment:
I'm not sure a 'move' is necessary since weight_nchw.vulkan() is already an r-value.